### PR TITLE
SKALE-779 Restructure SSL commands

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: node_cli/core/ssl/upload.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from node_cli.core.ssl.check import check_cert_openssl
+from node_cli.core.ssl.utils import is_ssl_folder_empty, copy_cert_key_pair
+from node_cli.utils.helper import ok_result
+
+logger = logging.getLogger(__name__)
+
+
+def upload_cert(cert_path, key_path, force, no_client=False):
+    try:
+        pass
```
Comment (badrogger) [inline node_cli/core/ssl/upload.py]: should be removed

File: node_cli/core/ssl/status.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+def cert_status():
+    if is_ssl_folder_empty():
+        return ok_result({'is_empty': True})
+
+    cert = cert_from_file(SSL_CERT_FILEPATH)
+    status, info = get_cert_info(cert)
+    if status == 'error':
+        return status, CERTS_INVALID_FORMAT
```
Comment (badrogger) [inline node_cli/core/ssl/status.py]: What about err_result ?

File: node_cli/core/ssl/upload.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        check_cert_openssl(
+            cert_path, key_path, silent=True, no_client=no_client)
+    except Exception as err:
+        logger.exception('Certificate/key pair is incorrect')
+        return 'error', f'Certificate check failed. {err}'
```
Comment (badrogger) [inline node_cli/core/ssl/upload.py]: it also can be err_result

File: tests/conftest.py [hunk 4]
```diff
@@ -185,3 +186,13 @@ def resource_alloc():
...
     os.remove(RESOURCE_ALLOCATION_FILEPATH)
+
+
+@pytest.fixture
+def ssl_folder():
+    if os.path.isdir(SSL_FOLDER_PATH):
+        shutil.rmtree(SSL_FOLDER_PATH)
+    path = pathlib.Path(SSL_FOLDER_PATH)
+    path.mkdir(parents=True, exist_ok=True)
+    yield
```
Comment (badrogger) [inline tests/conftest.py]: try finally is needed here

**Review by badrogger — CHANGES_REQUESTED** [review]

File: node_cli/core/ssl/upload.py (hunk 1 above)
Comment (dmytrotkk) [reply node_cli/core/ssl/upload.py]: ++

File: node_cli/core/ssl/status.py (hunk 2 above)
Comment (dmytrotkk) [reply node_cli/core/ssl/status.py]: ++

File: node_cli/core/ssl/upload.py (hunk 3 above)
Comment (dmytrotkk) [reply node_cli/core/ssl/upload.py]: ++

File: tests/conftest.py (hunk 4 above)
Comment (dmytrotkk) [reply tests/conftest.py]: ++

**Review by badrogger — APPROVED** [review]

**Pull request merged**